examples/tutorials/doc_examples/examples/event.py

```diff
@@ -10,10 +10,10 @@
 
 # event.py --- Example of a trait event
 
-# --<Imports>-------------------------------------------------------------------
+# --<Imports>------------------------------------------------------------------
```

Comment:
```suggestion
# --[Imports]------------------------------------------------------------------
```

Are these banners being used to parse the examples? If so, I hope the `<>` vs `[]` isn't significant. Otherwise, making this consistent with the other banners.

Reply:
Looks like the `<>` versus `[]` _is_ significant. :-(

https://github.com/enthought/traits/blob/a6411e971619b5566233be0f4e32e5e01bb37dbb/examples/tutorials/tutor.py#L104-L106

I changed at least one of these, assuming that there wasn't any significance. I'll change it back. (Though I also notice that *none* of these regexes will match at the moment, due to the extra space after the `#`, which was probably introduced as a result of a previous style cleanup.)

Reply:
Yep, I broke this when I applied `black` to the codebase in #432.